cli-commands/nodeos/subcommands/start/index.js

```diff
@@ -0,0 +1,62 @@
+const axios = require('axios');
+
+const Command = require('../../../command');
+const AsyncSoftExec = require('../../../helpers/async-soft-exec');
+
+const commandMessages = require('./messages');
+const commandExpression = require('./settings').command;
+const startCommandDefinition = require('./definition');
+
+// eoslime nodeos start --path
+
+class StartCommand extends Command {
+    constructor() {
+        super(startCommandDefinition);
+    }
+
+    async execute(args) {
+        try {
+            commandMessages.StartingNodeos();
+            const optionsResults = await super.processOptions(args);
+            await startNodeos(optionsResults.path);
+            return true;
+        } catch (error) {
+            commandMessages.UnsuccessfulStarting(error);
+        }
+        return false;
+    }
+}
+
+const startNodeos = async function(folderPath) {
+    const nodeosRunning = await isNodeosRunning();
+
+    if (nodeosRunning) {
+        commandMessages.NodeosAlreadyRunning();
+    } else {
+        await runNodeos(folderPath);
+    }
+}
+
+const isNodeosRunning = async function () {
```

Comment:
Check for pid file. By doing this, we will skip external libraries like axious. Fewer libs we use the better code support we will have.

If we did requests like the one below to check we are coupled to the API implementation, but it could change in the feature.

Reply:
axios removed and pid handling is used instead